sample-kotlin/src/main/java/com/fblitho/lithoktsample/lithography/data/DecadeViewModel.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ *  This file provided by Facebook is for non-commercial testing and evaluation
+ *  purposes only.  Facebook reserves all rights not expressly granted.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ *  FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ *  ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ *  CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+package com.fblitho.lithoktsample.lithography.data
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+
+class DecadeViewModel : ViewModel() {
```

Comment:
This can be a data class where model is a property with default value. That way anyone can mock-replace it for testing.

Reply:
Nice!